packages/models/fixtures/db.ts

```diff
@@ -102,6 +103,21 @@ export async function rule(payload?: Partial<IRule.CreatePayload>) {
   });
 }
 
+export async function slot(payload?: Partial<IAvailabilitySlot.CreatePayload>) {
+  const start = dayjs.utc(payload?.start || faker.date.future());
+  const end = start.add(faker.number.int(8), "hours");
+
+  return (
+    await availabilitySlots.create([
+      {
+        userId: await or.tutorId(payload?.userId),
+        start: start.toISOString(),
+        end: end.toISOString(),
+      },
+    ])
+  )[0];
```

Comment:
This code is not clean and prone to errors. I believe we can do better.

```suggestion
    const slots = await availabilitySlots.create([
      {
        userId: await or.tutorId(payload?.userId),
        start: start.toISOString(),
        end: end.toISOString(),
      },
    ])
    const slot = first(slots);
    if (!slot) throw new Error('Slot not found; should never happen');
    return slot;
```